derive_fuzztest_macro/src/lib.rs

```diff
@@ -348,9 +408,10 @@ mod tests {
                 extern crate std;
                 #[automatically_derived]
                 #[cfg(fuzzing)]
-                ::libfuzzer_sys::fuzz_target!(|args: (&[u8])| {
+                ::libfuzzer_sys::fuzz_target!(|args: (&[u8])| -> ::libfuzzer_sys::Corpus {
```

Comment:
since returning a TestResult is optional, is there still a test case that covers a generated fuzz_target! with no return type?

Reply:
Yes, the existing cases like https://github.com/google/rust-derive-fuzztest/blob/main/derive_fuzztest/fuzz/src/bin/arbitrary.rs still pass. The new generated code always returns a `Corpus::Keep` if there is no return value in the annotated function, matching the default in `fuzz_target!`.